app/services/idv/financials_step.rb

```diff
@@ -0,0 +1,32 @@
+module Idv
+  class FinancialsStep < Step
+    def complete?
+      idv_form.finance_type && idv_session.financials_confirmation.try(:success?) ? true : false
+    end
+
+    private
+
+    def vendor_validate
+      result = vendor_validator.validate
+      idv_session.params = idv_form.idv_params if complete?
+      result
+    end
+
+    def vendor_validator_class
+      Idv::FinancialsValidator
+    end
+
+    def analytics_event
+      Analytics::IDV_FINANCE_CONFIRMATION
+    end
+
+    def vendor_errors
+      idv_session.financials_confirmation.try(:errors)
```

Comment:
Using `try` is a sign that the code is not confident. Instead of not being sure if the session was updated, how about moving the errors to the FinancialsValidator class, where they belong IMO, since the errors come from the vendor. See https://github.com/18F/identity-idp/pull/947